addon/components/o-s-s/currency-input.ts

```diff
@@ -28,12 +29,18 @@ export default class OSSCurrencyInput extends Component<OSSCurrencyInputArgs> {
   }
 
   get selectedCurrencySymbol(): string {
-    return this.selectedCurrency.symbol || '—';
+    return this.selectedCurrency?.symbol || '—';
+  }
+
+  get selectedCurrencyCode(): string {
+    return this.selectedCurrency?.code || 'Select';
   }
 
   get selectedCurrency() {
-    if (isEmpty(this.args.currency)) {
+    if (isEmpty(this.args.currency) && this.args.onlyCurrency == false) {
```

Comment:
Thank you, i did that because the onlyCurrency can be undefined and  i thought that the typing of === could be a problem but not.
After some search on undifined status , i see that the problem is the logic of the function, i can simplify this function